eta-meeting-organizer-frontend/src/app/app.module.ts

```diff
@@ -8,12 +8,25 @@ import { TranslateHttpLoader } from '@ngx-translate/http-loader';
 import { SharedModule } from '~/app/shared/shared.module';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { AuthService, AuthServiceConfig } from "angularx-social-login";
+import { GoogleLoginProvider } from 'angularx-social-login';
 
 // http loader for translations file
 export function HttpLoaderFactory(http: HttpClient) {
   return new TranslateHttpLoader(http, '/assets/i18n/', '.json');
 }
 
+const config = new AuthServiceConfig([
+  {
+    id: GoogleLoginProvider.PROVIDER_ID,
+    provider: new GoogleLoginProvider('160652702041-14ipa76q95t63j3o974o3focpjsr51i7.apps.googleusercontent.com')
```

Comment:
erről nem beszéltünk benn, de jó lenne ha ez a string kikerülne az environment fájlba és a configuration service-en keresztül lenne hivva, nem lenne beégetve

Reply:
Javítva, betettem az environment objectbe.

Reply:
látom h valahol bevan huzva de itt pl még a string van, mindenhol dobd át pls

Reply:
Javítva.